novawallet/Modules/Settings/SettingsInteractor.swift

```diff
@@ -16,6 +16,9 @@ final class SettingsInteractor {
     let biometryAuth: BiometryAuthProtocol
     let walletConnect: WalletConnectDelegateInputProtocol
     let walletNotificationService: WalletNotificationServiceProtocol
+    let alertNotificationsService: Web3AlertsSyncServiceProtocol
+    let operationQueue: OperationQueue
+    let pushNotificationsService: PushNotificationsServiceProtocol
```

Comment:
As I understand push notification contains web3alert sync service. Why do we need both of them here?

Also we might pass factories or proxies instead of real services because we want to start services only if push notification enabled